--- e-uae-0.8.29-WIP4/src/hardfile.c.orig	2007-03-12 20:21:13.000000000 +0000
+++ e-uae-0.8.29-WIP4/src/hardfile.c	2008-03-21 02:10:49.000000000 +0000
@@ -432,8 +432,12 @@
     if (!hfpd->opencount)
 	return 0;
     hfpd->opencount--;
-    if (hfpd->opencount == 0)
+
+    if (hfpd->opencount == 0) {
 	write_comm_pipe_u32 (&hfpd->requests, 0, 1);
+	uae_sem_wait (&hfpd->sync_sem);
+    }
+
     put_word (m68k_areg (&context->regs, 6) + 32, get_word (m68k_areg (&context->regs, 6) + 32) - 1);
     return 0;
 }
@@ -663,7 +667,7 @@
 		error = handle_scsi (request, hfd);
 	    } else { /* we don't want users trashing their "partition" hardfiles with hdtoolbox */
 		error = -3; /* IOERR_NOCMD */
-		write_log ("UAEHF: HD_SCSICMD tried on regular HDF, unit %d", unit);
+		write_log ("UAEHF: HD_SCSICMD tried on regular HDF, unit %d\n", unit);
 	    }
 	break;
 
@@ -793,11 +797,20 @@
 		if ((request = hfpd->d_request[i]))
 		    abort_async (hfpd, request, 0, 0);
 	    }
+
+	    write_comm_pipe_u32 (&hfpd->requests, 0, 1);
+	    uae_sem_wait (&hfpd->sync_sem);
 	}
+
 	memset (hfpd, 0, sizeof (struct hardfileprivdata));
     }
 }
 
+void hardfile_cleanup (void)
+{
+    hardfile_reset ();
+}
+
 void hardfile_install (void)
 {
     uae_u32 functable, datatable;
